cli/output/rpc_progress.go

```diff
@@ -40,6 +41,21 @@ func ProgressBar() rpc.DownloadProgressCB {
 	}
 }
 
+// PrintErrorFromDownloadResult returns a DownloadResultCB that only prints
+// the errors with the give message prefixed.
+func PrintErrorFromDownloadResult(msg string) rpc.DownloadResultCB {
+	if OutputFormat != "json" {
```

Comment:
The `json` output format should not be given special treatment. All the machine readable output formats (`json`, `jsonmini`, `yaml`) must be given equal consideration in this type of code. 

There should be a single centralized function to handle this sort of thing instead of unmaintained "magic strings" scattered throughout the codebase.

Reply:
You're right, but this is outside the scope of this specific PR, I'll open a new issue for this one.